public/expenses/edit.php

```diff
@@ -93,10 +96,10 @@ class='expense__form--content-input'
                 </div>
                 <hr>
                 <!-- Submit button -->
-                <div class='expense__form--content'>
-                    <div>
+                <div >
+                    <div class="expense__form--btn-submit">
                         <button type="submit" name="submit" value="submit" class="btn btn-primary">
-                            Update
+                            Submit
```

Comment:
Submit is a general action and "update" was meant here to inform the user that he is updating an existing record.